src/webgpu/api/validation/encoding/cmds/render/draw.spec.ts

```diff
@@ -7,120 +7,45 @@ and parameters as expect.
 import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
 import { ValidationTest } from '../../../validation_test.js';
 
-// Class that indicate how to call a draw function
-class DrawCall {
-  test: ValidationTest;
-  drawType: 'draw' | 'drawIndexed' | 'drawIndirect' | 'drawIndexedIndirect' = 'draw';
-
-  // Draw
-  vertexCount: number = 4;
-  firstVertex: number = 0;
-
-  // DrawIndexed
-  indexCount: number = 4;
-  firstIndex: number = 0;
-  baseVertex: number = 0;
-
-  // Both Draw and DrawIndexed
-  instanceCount: number = 1;
-  firstInstance: number = 0;
-
-  constructor(test: ValidationTest) {
-    this.test = test;
-  }
+interface DrawIndexedParameter {
+  indexCount: number;
+  firstIndex?: number;
+  baseVertex?: number;
+  instanceCount?: number;
+  firstInstance?: number;
```

Comment:
uNit: please order these the same way as in the drawIndexed call.

Reply:
Done.